translate/compiler_test.go

```diff
@@ -16,9 +16,82 @@
 
 package translate_test
 
-import "testing"
+import (
+	"testing"
+
+	"github.com/DE-labtory/koa/abi"
+	"github.com/DE-labtory/koa/ast"
+	"github.com/DE-labtory/koa/translate"
+)
+
+func TestFuncMap_Declare(t *testing.T) {
```

Comment:
plz add testcase when there are two more functions.

Reply:
For `Declare`? This function handles just one `FunctionLiteral`. And, test `bar` and `sam` declares when there are two or more functions.